proxy/src/schema/error.rs

```diff
@@ -297,6 +308,9 @@ fn convert_url_parse_error_to_field_error(error: &url::ParseError) -> FieldError
 impl IntoFieldError for Error {
     fn into_field_error(self) -> FieldError {
         match self {
+            Self::Cbor(cbor_error) => {
```

Comment:
This is the first time I think we are facing a range of errors which are not meaningful to the upstream layers. We should rethink what our strategy when we encounter a serialisation error. And in turn what we really want to communicate potentially to the end user. A cbor error won't be actionable.

Reply:
Interesting. I suppose an `.expect` might be appropriate then? Or falling back to an empty vec?